### ISSUE TITLE
webpack: assets should be namespaced

### ISSUE THREAD

**Issue by slint** [inveniosoftware/invenio-previewer#108]
Since all webpack assets end up in the same folder (when you run `invenio webpack create`), an undocumented "best practice" is to have a namespaced directory structure:

```
invenio_<MODULE>/assets
├── css
│   └── invenio_<MODULE>
│       └── styles.scss
└── js
    └── invenio_<MODULE>
        ├── index.js
        ├── forms.js
        └── search.js
```

**Comment by topless** [comment]
For future consideration, maybe we could move the namespacing responsibility to from each module to `invenio-assets`.

**Comment by ppanero** [comment]
closed by #134 (superseded #132)
